compiler/extccomp.nim

```diff
@@ -26,8 +26,10 @@ type
     hasAssume,                # CC has __assume (Visual C extension)
     hasGcGuard,               # CC supports GC_GUARD to keep stack roots
     hasGnuAsm,                # CC's asm uses the absurd GNU assembler syntax
-    hasNakedDeclspec,         # CC has __declspec(naked)
-    hasNakedAttribute         # CC has __attribute__((naked))
+    hasDeclspec,              # CC has __declspec(X)
+    hasAttribute,             # CC has __attribute__((X))
+    hasNaked,                 # CC has naked declspec/attribute
+    hasNoreturn               # CC has noreturn declspec/attribute
```

Comment:
That's not what I meant. Assume that ``hasDeclspec`` implies that both ``__declspec(naked)`` and ``__declspec(noreturn)`` are available. Likewise for ``hasAttribute``. I don't think there are compilers where this assumption does not hold.